### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by dianajodrakos**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -8,22 +8,43 @@ const input3 = document.querySelector('#poke-3');
...
 // initialize state
 renderThreePokemon();
 
 // set event listeners 
-button.addEventListener('click', () => {
+catchButton.addEventListener('click', () => {
   // get user input
-  const input = document.querySelector('input[type=radio]:checked');
+  let input = document.querySelector('input[type=radio]:checked');
+  if (input === null) {
```
Comment (breeannb) [inline app.js:21]: Love that you added some error handling when a user hasn't clicked a radio button.  There are also ways, like putting ```checked``` on html or some other javascript logic that can make sure a radio button is always selected as well!

File: alltime.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            label: 'Pokémon caught',
+            data: capturedData,
+            backgroundColor: capturedColorsData,
+            borderColor: 'grey',
+            borderWidth: 2
+        }]
+    }
+});
+
+var ctx2 = document.getElementById('encountered-chart').getContext('2d');
```
Comment (breeannb) [inline alltime.js:29]: nice, I love the different types of charts you have.  Looks great!

File: local-storage-utils.js [hunk 3]
```diff
@@ -9,12 +10,26 @@ export function getPokedex() {
...
-};
+}
 
 export function setPokedex(pokedex) {
     const stringyPokedex = JSON.stringify(pokedex);
     localStorage.setItem(POKEDEX, stringyPokedex);
-};
+}
+
+export function getSessions() {
```
Comment (breeannb) [inline local-storage-utils.js:20]: sessions page wasn't working quite yet, and the redirect for a ```New Session``` also wasn't re-routing on your deployed.   Might be something to check out if you have time to work on this more.

**Review by dianajodrakos — COMMENTED** [review]
test comment, not referencing a single line

**Pull request merged**